src/vehicle.cpp

```diff
@@ -4118,10 +4150,19 @@ int vehicle::max_rotor_velocity( const bool fueled ) const
     return std::min( 25501, mps_to_vmiph( max_air_mps ) );
 }
 
+int vehicle::max_air_velocity( const bool fueled ) const
+{
+    const double max_air_mps = std::sqrt( lift_thrust_of_abstracted_aircraft( fueled ) / coeff_air_drag() );
```

Comment:
Why are "abstracted aircraft" limited by the physics of rotors, which they are not using?

Reply:
Because it appeared necessary to add a movement mode to allow something without rotors to fly due to the amount of checks currrently preventing anything without rotors from flying.

Turned out that the check for needing wheels to move and the check to move while airborne don't conflict with each other, so I'll remove it soon. Thanks for pointing that out.

As a bonus, it also turned out that there's no need for helicopters to check if they have enough rotorlift to use autodrive, as they cannot use autodrive unless airborne and the check forbids other vehicles from using it while airborne, so I'll remove that extraneous autodrive check too as they lack the wheels to move without flying.